modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -27,16 +27,16 @@ function acquia_cms_common_install($is_syncing) {
       'view media',
     ]);
 
-    // Enable acquia_claro theme.
+    // Enable gin theme.
     $themesList = \Drupal::service('extension.list.theme')->getList();
     $themesInstall = ["olivero"];
-    if (isset($themesList['acquia_claro'])) {
-      $themesInstall[] = "acquia_claro";
+    if (isset($themesList['gin'])) {
+      $themesInstall[] = "gin";
     }
     \Drupal::service('theme_installer')->install($themesInstall);
     \Drupal::configFactory()->getEditable('system.theme')->set('default', 'olivero')->save();
-    if (in_array("acquia_claro", $themesInstall)) {
-      \Drupal::configFactory()->getEditable('system.theme')->set('admin', "acquia_claro")->save();
+    if (in_array("gin", $themesInstall)) {
+      \Drupal::configFactory()->getEditable('system.theme')->set('admin', "gin")->save();
```

Comment:
Can we do something like this?
$system_config_factory = \Drupal::configFactory()->getEditable('system.theme');
$system_config_factory->set('default', 'olivero')->save();
if (in_array("gin", $themes_install)) {
$system_config_factory->set('admin', 'gin')->save();
}